OpenInActionExtension/Media.xcassets/Icon.appiconset/Contents.json

```diff
@@ -0,0 +1,14 @@
+{
+  "images" : [
+    {
+      "filename" : "MastodonActionExtensionIcon@3x.png",
```

Comment:
What's the reason for `.png` instead of `.pdf` or another vector-graphic-format?

Reply:
Ah, nevermind, just found the archive